### PR TITLE
chore: add changelog

### PR DESCRIPTION
**Pull request by jeremad**

### CODE REVIEW

File: Changelog.rst [hunk 1]
```diff
@@ -1,3 +1,15 @@
+3.0.0
+=====
+
+Please read the `migration guide <https://docs.tanker.io/latest/release-notes/identity/go/guide/>`_ before upgrading to this version if you use the provisional identity feature.
```
Comment (tux3) [inline Changelog.rst:4]: This should probably link to `https://docs.tanker.io/latest/release-notes/identity/python/guide/` instead of /go/

**Review by tux3 — APPROVED** [review]
Otherwise LGTM

File: Changelog.rst (hunk 1 above)
Comment (jeremad) [reply Changelog.rst:4]: oops

**Comment by tux3** [comment]
Changelog :+1:

**Pull request merged**